script/Ciel/Day 13/13_01_01_CIEL13_1_1.txt

```diff
@@ -246,7 +246,9 @@ Instead, it had the opposite effect.
 [sha:ff7ca42e9eb06e0e9544ccb9b390f7d57266f868]{
 -- Page 64, Offset 33308.
 -- ■■は安心しきって、ベッドにうずくまった俺の／オレの、体に／躯に、顔を／貌を、寄せて／捧げて、きた。
-Feeling at ease, ■■ draws her face against my crouched-up body.
+// is the ／ part of the line somewhere else or did tumbling just forget
```

Comment:
they're incredibly clunky, but we can leave these for the guys in dover to deal with